scss/main.scss

```diff
@@ -1,3 +1,4 @@
 @import 'reset';
 @import 'navigation';
 @import 'footer';
+@import 'validation'
```

Comment:
Should _validation.scss be in main.scss? Maybe it can be put in there but not linked to main.scss, the forms that need them can just pull from it using @import '../validation' instead.